src/main/java/uk/gov/justice/hmpps/prison/values/Money.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.hmpps.prison.values;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class Money {
```

Comment:
Seems strange for us to be modeling stuff outside our domain is there something in the java lib we could use for this?

Reply:
as I understanding it entryAmount is money value and that needs to be represented as pence value as discussed previously and that conversion code needs to be placed somewhere, please suggest a better approach if you wish

Reply:
I've seen it done this way in most places, in pounds but seems like a simple solution -   

```
public BigDecimal getAmountInPounds() {
        return new BigDecimal(amount).setScale(2, RoundingMode.HALF_UP).divide(new BigDecimal("100"), RoundingMode.HALF_UP);
    }
```

Reply:
I'd say just return the amounts as they come (pounds). Unless there a problem we're trying to solve by returning pence.

Reply:
i assumed from exiting code that we represent money in pence value, if that's not the case we don't need this